core/slim_globals.h

```diff
@@ -352,6 +352,7 @@ extern EidosValue_String_SP gStaticEidosValue_StringT;
 	
 #define SLIM_TSK_INDIVIDUAL_ALIVE       ((uint32_t)(1 << 16))
 #define SLIM_TSK_INDIVIDUAL_REMEMBERED  ((uint32_t)(1 << 17))
+// DEPRECATED:
 #define SLIM_TSK_INDIVIDUAL_FIRST_GEN   ((uint32_t)(1 << 18))
```

Comment:
If we're no longer using this anywhere in the code, can we actually comment this line out, to ensure that nobody uses it?

Reply:
Yes, I guess so - I thought we'd need to keep it around to load old files, but since we don't...